userspace/libsinsp/utils.h

```diff
@@ -184,6 +184,28 @@ inline void sanitize_string(std::string &str)
 	str.erase(remove_if(str.begin(), str.end(), g_invalidchar()), str.end());
 }
 
+inline void remove_duplicate_path_separators(std::string &str)
+{
+	// Light fd name sanitization if fd is a file - only remove consecutive duplicate separators
+	char prev_char;
+	char separator = '/';
+	for (auto cur_char_it = str.begin(); cur_char_it != str.end(); cur_char_it++)
+	{
+		if (prev_char == *cur_char_it)
+		{
+			if (prev_char == separator)
+			{
+				str.erase(cur_char_it);
+				cur_char_it--;
```

Comment:
I think the `erase` function could potentially invalidate the iterator, so maybe we want to reassign it using the return value, which for sure is a valid one: `cur_chan_it = str.erase(cur_chan_it)`. What do you think?

Reply:
+1 on this.